realm/realm-jni/build.gradle

```diff
@@ -248,6 +248,109 @@ task deployCore(group: 'build setup', description: 'Deploy the latest version of
     }
 }
 
+task compileCoreLibForHost(group: 'build setup', description: 'Compile the core library for host from source code') {
+    doLast {
+        if (!coreSourcePath) {
+            throw new GradleException("The Realm Core library is needed to build support for Robolectric. coreSourcePath, /usr/local/lib/librealm.a or the environment REALM_CORE_HOME is needed.")
+        }
+        exec {
+            workingDir = coreSourcePath
+            environment 'REALM_ENABLE_ENCRYPTION', 'yes'
```

Comment:
Since itts build runs without REALM_ANDROID, REALM_ENABLE_ASSERTIONS is needed I think. 
See also: https://github.com/realm/realm-java/pull/1867/files#diff-79341f7ddca6be6ffebaed37d9e394b5L40